### PR TITLE
fix(io.ipset): raise exception if entries exceed limit

### LINKED ISSUE #1280: ipset: config checks should verify that the number of elements does not exceed ipset size
### What happened

Fedora bug: https://bugzilla.redhat.com/show_bug.cgi?id=2258244

### What you expected to happen

firewall-offline-cmd --check-config should complain if there are more elements than "size" allows.

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Anything else we need to know?

_No response_

### Firewalld Version

1.3.4

### Firewalld Backend

_No response_

### Linux distribution

_No response_

### Linux kernel version

_No response_

### Other information

_No response_

### PR DESCRIPTION
**Pull request by TorontoMedia**
Closes #1280

### CODE REVIEW

File: src/firewall/core/io/ipset.py [hunk 1]
```diff
@@ -430,6 +430,16 @@ def ipset_reader(filename, path):
...
+
+    if "maxelem" in ipset.options and len(ipset.entries) > int(
+        ipset.options["maxelem"]
+    ):
+        raise FirewallError(
+            errors.INVALID_IPSET,
+            "the number of ipset entries exceeds the maximum set size of %s"
+            % ipset.options["maxelem"],
+        )
+
```
Comment (erig0) [inline src/firewall/core/io/ipset.py:442]: I think this more correctly belongs in `_check_config()`.

**Review by erig0 — CHANGES_REQUESTED** [review]
Otherwise, lgtm.

File: src/firewall/core/io/ipset.py (hunk 1 above)
Comment (TorontoMedia) [reply src/firewall/core/io/ipset.py]: You are correct. I have made the necessary changes and modified gh881 maximum element limit.

**Review by erig0 — APPROVED** [review]
Thank you!!

**Pull request merged**